### DISCUSSION TITLE
What is the function of the "default component list"?

### DISCUSSION THREAD

**Discussion by tomkolp** [WeblateOrg/weblate#5593] [General] [answered]
I thought that after selecting the "default component list" it would be loaded on the dashboard.  When I choose any list, the dashboard is completely empty.
![Screenshot_20210305-182218](https://user-images.githubusercontent.com/39633242/110151466-cf5c2180-7de0-11eb-86ef-2a4ec976e495.png)
![Screenshot_20210305-182912](https://user-images.githubusercontent.com/39633242/110151480-d4b96c00-7de0-11eb-9d8c-575f26584343.png)

**Comment by nijel** [comment, marked as answer]
It allows you to choose a component list to display as default view on the dashboard. There is currently a bug that allows choosing even the component lists which are not configured to be shown on the dashboard. This is fixed in 5f3a23da4da6a9d222b0cd98811c72dce2642e09.

**Reply by tomkolp** [reply]
How do I hide a screenshot while creating a post? Are they terribly large and unreadable from the phone? @nijel

**Reply by comradekingu** [reply]
@tomkolp 
Unless you go to preview that shouldn't happen. I just see
`![image](https://user-images.githubusercontent.com/imagename.png)`
unless I go to preview if I just paste the image.

Try setting your mobile browser to desktop mode?

**Reply by tomkolp** [reply]
[5f3a23d](https://github.com/WeblateOrg/weblate/commit/5f3a23da4da6a9d222b0cd98811c72dce2642e09)  after this change, the list of components to choose from is empty
![110199512-00783880-7e59-11eb-97d9-baa792da973a](https://user-images.githubusercontent.com/39633242/110249186-3dd1e880-7f75-11eb-8ae5-27379ddd43b5.png)

**Reply by nijel** [reply]
The list now shows only component lists configured to show on the dashboard and where user has access to at least one of the components. Most likely your component lists are not configured to be shown on the dashboard.

**Reply by nijel** [reply]
I've changed it now to show only single selection here: 1a65fd0dc40afd0672a7e6e2765fe968ae1006e3
